service/frontend/workflowHandler.go

```diff
@@ -3512,6 +3512,21 @@ func (wh *WorkflowHandler) ReapplyEvents(
 	return nil
 }
 
+// GetCadenceClusterInfo return information about cadence deployment
+func (wh *WorkflowHandler) GetCadenceClusterInfo(
+	ctx context.Context,
+) (resp *gen.CadenceClusterInfo, err error) {
+	defer log.CapturePanic(wh.GetLogger(), &err)
+
```

Comment:
also include the throttling

Reply:
I didn't understand what you meant by including the throttling. Can you point me to an example?